<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" source-language="en" original="../Resource.resx" target-language="es">
    <body>
      <trans-unit id="Execution_Test_Timeout">
        <source>Test '{0}' exceeded execution timeout period.</source>
        <target state="translated">La prueba '{0}' superó el tiempo de espera de ejecución.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="SourcesNotSupported">
        <source>Running tests in any of the provided sources is not supported for the selected platform</source>
        <target state="translated">La ejecución de pruebas en las fuentes proporcionadas no se admite para la plataforma seleccionada</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_CleanupMethodThrows">
        <source>TestCleanup method {0}.{1} threw exception. {2}.</source>
        <target state="translated">El método TestCleanup {0}.{1} devolvió una excepción. {2}.</target>
        <note />
      </trans-unit>
      <trans-unit id="UTA_EndOfInnerExceptionTrace">
        <source>--- End of inner exception stack trace ---</source>
        <target state="translated">--- Fin del seguimiento de la pila de excepción interna ---</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorMultiAssemblyClean">
        <source>UTA014: {0}: Cannot define more than one method with the AssemblyCleanup attribute inside an assembly.</source>
        <target state="translated">UTA014: {0}: No se puede definir más de un método con el atributo AssemblyCleanup dentro de un ensamblado.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorMultiAssemblyInit">
        <source>UTA013: {0}: Cannot define more than one method with the AssemblyInitialize attribute inside an assembly.</source>
        <target state="translated">UTA013: {0}: No se puede definir más de un método con el atributo AssemblyInitialize dentro de un ensamblado.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorMultiClassClean">
        <source>UTA026: {0}: Cannot define more than one method with the ClassCleanup attribute inside a class.</source>
        <target state="translated">UTA026: {0}: No se puede definir más de un método con el atributo ClassCleanup dentro de una clase.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorMultiClassInit">
        <source>UTA025: {0}: Cannot define more than one method with the ClassInitialize attribute inside a class.</source>
        <target state="translated">UTA025: {0}: No se puede definir más de un método con el atributo ClassInitialize dentro de una clase.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorMultiClean">
        <source>UTA024: {0}: Cannot define more than one method with the TestCleanup attribute.</source>
        <target state="translated">UTA024: {0}: No se puede definir más de un método con el atributo TestCleanup.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorMultiInit">
        <source>UTA018: {0}: Cannot define more than one method with the TestInitialize attribute.</source>
        <target state="translated">UTA018: {0}: No se puede definir más de un método con el atributo TestInitialize.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_InitMethodThrows">
        <source>Initialization method {0}.{1} threw exception. {2}.</source>
        <target state="translated">El método de inicialización {0}.{1} devolvió una excepción. {2}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_InstanceCreationError">
        <source>Unable to create instance of class {0}. Error: {1}.</source>
        <target state="translated">No se puede crear una instancia de la clase {0}. Error: {1}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_TestContextSetError">
        <source>Unable to set TestContext property for the class {0}. Error: {1}.</source>
        <target state="translated">No se puede establecer la propiedad TestContext para la clase {0}. Error: {1}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTF_FailedToGetExceptionMessage">
        <source>(Failed to get the message for an exception of type {0} due to an exception.)</source>
        <target state="translated">(No se pudo obtener el mensaje para una excepción del tipo {0} debido a una excepción.)</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorInValidTestContextSignature">
        <source>UTA031: class {0} does not have valid TestContext property. TestContext must be of type TestContext, must be non-static, public and must not be read-only. For example: public TestContext TestContext.</source>
        <target state="translated">UTA031: la clase {0} no tiene la propiedad TestContext válida. TestContext debe ser de tipo TestContext, debe ser no estática, pública y no debe ser de solo lectura. Por ejemplo: public TestContext TestContext.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorNonPublicTestClass">
        <source>UTA001: TestClass attribute defined on non-public class {0}</source>
        <target state="translated">UTA001: se ha definido el atributo TestClass en la clase no pública {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="CouldNotInspectTypeDuringDiscovery">
        <source>MSTestAdapter failed to discover tests in class '{0}' of assembly '{1}' because {2}.</source>
        <target state="translated">MSTestAdapter no detectó pruebas en la clase '{0}' del ensamblado '{1}' porque {2}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="EnumeratorLoadTypeErrorFormat">
        <source>{0}: {1}</source>
        <target state="translated">{0}: {1}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TypeLoadFailed">
        <source>Unable to load types from the test source '{0}'. Some or all of the tests in this source may not be discovered.
Error: {1}</source>
        <target state="translated">No se pueden cargar tipos del origen de prueba "{0}". Puede que no se detecten algunas o ninguna de las pruebas de este origen.
Error: {1}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorGenericTestMethod">
        <source>UTA015: A generic method cannot be a test method. {0}.{1} has invalid signature</source>
        <target state="translated">UTA015: Un método genérico no puede ser un método de prueba. {0}.{1} tiene una firma no válida</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestAssembly_FileDoesNotExist">
        <source>File does not exist: {0}</source>
        <target state="translated">El archivo no existe: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorIncorrectTestMethodSignature">
        <source>UTA007: Method {1} defined in class {0} does not have correct signature. Test method marked with the [TestMethod] attribute must be non-static, public, return-type as void  and should not take any parameter. Example: public void Test.Class1.Test(). Additionally, if you are using async-await in test method then return-type must be Task. Example: public async Task Test.Class1.Test2()</source>
        <target state="translated">UTA007: El método {1} definido en la clase {0} no tiene una signatura correcta. El método de prueba marcado con el atributo [TestMethod] debe ser no estático, público, con el tipo devuelto void y no debe tomar ningún parámetro. Ejemplo: public void Test.Class1.Test(). Además, si está usando async-await en el método de prueba, el tipo devuelto debe ser Task. Ejemplo: public async Task Test.Class1.Test2()</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestContextIsNull">
        <source>TestContext cannot be Null.</source>
        <target state="translated">TestContext no será null.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_AssemblyCleanupMethodWasUnsuccesful">
        <source>Assembly Cleanup method {0}.{1} failed. Error Message: {2}. StackTrace: {3}</source>
        <target state="translated">Error de método Cleanup de ensamblado {0}.{1}. Mensaje de error: {2}. StackTrace: {3}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_AssemblyInitMethodThrows">
        <source>Assembly Initialization method {0}.{1} threw exception. {2}: {3}. Aborting test execution.</source>
        <target state="translated">Excepción método inicialización ensamblado {0}.{1}. {2}: {3}. Anulada ejecución de prueba.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ClassCleanupMethodWasUnsuccesful">
        <source>Class Cleanup method {0}.{1} failed. Error Message: {2}. Stack Trace: {3}</source>
        <target state="translated">Error de método Cleanup de clase {0}.{1}. Mensaje error: {2}. Seguimiento de pila: {3}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ClassInitMethodThrows">
        <source>Class Initialization method {0}.{1} threw exception. {2}: {3}.</source>
        <target state="translated">Excepción del método inicialización clase {0}. {1}. {2}: {3}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ExecuteThrewException">
        <source>Exception thrown while executing test. If using extension of TestMethodAttribute then please contact vendor. Error message: {0}, Stack trace: {1}</source>
        <target state="translated">Se inició una excepción al ejecutar la prueba. Si usa la extensión de TestMethodAttribute, póngase en contacto con el proveedor. Mensaje de error: {0}, seguimiento: {1}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_NoTestResult">
        <source>Error in executing test. No result returned by extension. If using extension of TestMethodAttribute then please contact vendor.</source>
        <target state="translated">No se pudo ejecutar prueba. Extensión no devolvió resultados. Si usa extensión TestMethodAttribute, contacte con el proveedor.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ClassOrAssemblyCleanupMethodHasWrongSignature">
        <source>Method {0}.{1} has wrong signature. The method must be static, public, does not return a value and should not take any parameter. Additionally, if you are using async-await in method then return-type must be Task.</source>
        <target state="translated">Método {0}.{1} tiene firma incorrecta. El método debe ser estático, público, no devolver un valor y no aceptar ningún parámetro. Además, si usa async-esperan en método de tipo de valor devuelto debe ser una tarea.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ClassOrAssemblyInitializeMethodHasWrongSignature">
        <source>Method {0}.{1} has wrong signature. The method must be static, public, does not return a value and should take a single parameter of type TestContext. Additionally, if you are using async-await in method then return-type must be Task.</source>
        <target state="translated">Método {0}.{1} tiene firma incorrecta. El método debe ser estático y público, no devolver ningún valor y debe aceptar un solo parámetro tipo TestContext. Además, si está usando async-await en el método, el tipo devuelto debe ser Task.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorInvalidTimeout">
        <source>UTA054: {0}.{1} has invalid Timeout attribute. The timeout must be a valid integer value and cannot be less than 0.</source>
        <target state="translated">UTA054: atributo tiempo espera {0}.{1} no válido. Valor tiempo de espera debe ser entero válido y no menor que 0.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorPredefinedTestProperty">
        <source>UTA023: {0}: Cannot define predefined property {2} on method {1}.</source>
        <target state="translated">UTA023: {0}: no se puede definir la propiedad predefinida {2} en el método {1}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ErrorTestPropertyNullOrEmpty">
        <source>UTA021: {0}: Null or empty custom property defined on method {1}. The custom property must have a valid name.</source>
        <target state="translated">UTA021: {0}: se ha definido una propiedad personalizada nula o vacía en el método {1}. La propiedad personalizada debe tener un nombre válido.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_MethodDoesNotExists">
        <source>Method {0}.{1} does not exist.</source>
        <target state="translated">El método {0}.{1} no existe.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_NoDefaultConstructor">
        <source>Unable to get default constructor for class {0}.</source>
        <target state="translated">No se puede obtener el constructor predeterminado para la clase {0}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_TestContextLoadError">
        <source>Unable to find property {0}.TestContext. Error:{1}.</source>
        <target state="translated">No se puede encontrar la propiedad {0}.TestContext. Error:{1}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_TestContextTypeMismatchLoadError">
        <source>The {0}.TestContext has incorrect type.</source>
        <target state="translated">Tipo {0}.TestContext no es correcto.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_TestInitializeAndCleanupMethodHasWrongSignature">
        <source>Method {0}.{1} has wrong signature. The method must be non-static, public, does not return a value and should not take any parameter. Additionally, if you are using async-await in method then return-type must be Task.</source>
        <target state="translated">Método {0}.{1} tiene firma incorrecta. Debe ser un método no estático, público, no devolver ningún valor y no debe aceptar parámetros. Además, si está usando async-await en el método, el tipo devuelto debe ser Task.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_TypeLoadError">
        <source>Unable to get type {0}. Error: {1}.</source>
        <target state="translated">No se puede obtener el tipo {0}. Error: {1}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestNotFound">
        <source>Test method {0} was not found.</source>
        <target state="translated">No encontrado método prueba {0}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="DebugTraceBanner">
        <source>Debug Trace:</source>
        <target state="translated">Seguimiento de depuración:</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_FailedToGetTestMethodException">
        <source>Failed to obtain the exception thrown by test method {0}.{1}.</source>
        <target state="translated">No se pudo obtener la excepción iniciada por el método de prueba {0}.{1}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_TestMethodThrows">
        <source>Test method {0}.{1} threw exception: 
{2}</source>
        <target state="translated">Excepción método de prueba {0}.{1}: 
{2}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_WrongThread">
        <source>{0} For UWP projects, if you are using UI objects in test consider using [UITestMethod] attribute instead of [TestMethod] to execute test in UI thread.</source>
        <target state="translated">{0}. En proyectos de UWP, si usa objetos de interfaz de usuario en la prueba, podría usar el atributo [UITestMethod] en lugar de [TestMethod] para ejecutar la prueba en el subproceso de interfaz de usuario.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="AttachmentSetDisplayName">
        <source>MSTestAdapterV2</source>
        <target state="translated">MSTestAdapterV2</target>
        <note></note>
      </trans-unit>
      <trans-unit id="InvalidSettingsXmlAttribute">
        <source>Invalid settings '{0}'. Unexpected XmlAttribute: '{1}'.</source>
        <target state="translated">Valor no válido '{0}'. XmlAttribute no esperado: '{1}'.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="InvalidSettingsXmlElement">
        <source>Invalid settings '{0}'. Unexpected XmlElement: '{1}'.</source>
        <target state="translated">Valor no válido '{0}'. XmlElement no esperado: '{1}'.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="DataDrivenResultDisplayName">
        <source>{0} (Data Row {1})</source>
        <target state="translated">{0} (Fila de datos {1})</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_ExpectedExceptionAttributeConstructionException">
        <source>The ExpectedException attribute defined on test method {0}.{1} threw an exception during construction.
{2}</source>
        <target state="translated">El atributo ExpectedException definido en el método de prueba {0}.{1} inició una excepción durante la construcción.
{2}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_MultipleExpectedExceptionsOnTestMethod">
        <source>The test method {0}.{1} has multiple attributes derived from ExpectedExceptionBaseAttribute defined on it. Only one such attribute is allowed.</source>
        <target state="translated">El método de prueba {0}.{1} tiene definidos varios atributos derivados de ExpectedExceptionBaseAttribute. Solo se permite uno de estos atributos.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="LegacyScenariosNotSupportedWarning">
        <source>Warning : A testsettings file or a vsmdi file is not supported with the MSTest V2 Adapter.</source>
        <target state="translated">Advertencia: No se admite un archivo testsettings o vsmdi con el adaptador de MSTest V2.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestContextMessageBanner">
        <source>TestContext Messages:</source>
        <target state="translated">Mensajes de TestContext:</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_CleanupMethodThrowsGeneralError">
        <source>Error calling Test Cleanup method for test class {0}: {1}</source>
        <target state="translated">Error al llamar al método Test Cleanup para la clase de prueba {0}: {1}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_CleanupStackTrace">
        <source>TestCleanup Stack Trace</source>
        <target state="translated">Seguimiento de pila de TestCleanup</target>
        <note></note>
      </trans-unit>
      <trans-unit id="DiscoveryWarning">
        <source>[MSTest][Discovery][{0}] {1}</source>
        <target state="translated">[MSTest][Discovery][{0}] {1}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestParallelizationBanner">
        <source>MSTest Executor: Test Parallelization enabled for {0} (Workers: {1}, Scope: {2}).</source>
        <target state="translated">Ejecutor de MSTest Executor: paralelización de prueba habilitada para {0} (Trabajadores: {1}, Ámbito: {2}).</target>
        <note></note>
      </trans-unit>
      <trans-unit id="InvalidParallelScopeValue">
        <source>Invalid value '{0}' specified for 'Scope'. Supported scopes are {1}.</source>
        <target state="translated">Valor no válido "{0}" especificado para "Ámbito". Los ámbitos admitidos son {1}.</target>
        <note>'Scope' is a setting name that shouldn't be localized.</note>
      </trans-unit>
      <trans-unit id="InvalidParallelWorkersValue">
        <source>Invalid value '{0}' specified for 'Workers'. The value should be a non-negative integer.</source>
        <target state="translated">Valor no válido "{0}" especificado para "Trabajadores". El valor debe ser un entero no negativo.</target>
        <note>`Workers` is a setting name that shouldn't be localized.</note>
      </trans-unit>
      <trans-unit id="TestAssembly_AssemblyDiscoveryFailure">
        <source>Failed to discover tests from assembly {0}. Reason:{1}</source>
        <target state="translated">No se pudieron detectar pruebas desde el ensamblado {0}. Motivo:{1}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_TestMethodExpectedParameters">
        <source>Only data driven test methods can have parameters. Did you intend to use [DataRow] or [DynamicData]?</source>
        <target state="translated">Solo los métodos de pruebas controladas por datos pueden tener parámetros. ¿Quería usar [DataRow] o [DynamicData]?</target>
        <note></note>
      </trans-unit>
      <trans-unit id="Execution_Test_Cancelled">
        <source>Test '{0}' execution has been aborted.</source>
        <target state="translated">Se anuló la ejecución de la prueba "{0}".</target>
        <note></note>
      </trans-unit>
      <trans-unit id="InvalidClassCleanupLifecycleValue">
        <source>Invalid value '{0}' specified for 'ClassCleanupLifecycle'. Supported scopes are {1}.</source>
        <target state="translated">Valor no válido "{0}" especificado para "ClassCleanupLifecycle". Los ámbitos admitidos son {1}.</target>
        <note>'ClassCleanupLifecycle' is a setting name that shouldn't be localized.</note>
      </trans-unit>
      <trans-unit id="CannotEnumerateIDataSourceAttribute">
        <source>Exception occurred while enumerating IDataSource attribute on "{0}.{1}": {2}</source>
        <target state="translated">Excepción al enumerar el atributo IDataSource en "{0}.{1}": {2}</target>
        <note>{0}: TypeName with namespace,
{1}: Method name,
{2}: Exception details</note>
      </trans-unit>
      <trans-unit id="ExceptionOccuredWhileGettingTheExceptionDescription">
        <source>"{0}": (Failed to get exception description due to an exception of type "{1}".</source>
        <target state="translated">"{0}": (No se pudo obtener la descripción de la excepción debido a una excepción de tipo "{1}".</target>
        <note>{0}: Type of the original exception that we're trying to get the description of.
{1}: Thrown exception</note>
      </trans-unit>
      <trans-unit id="ExceptionsThrown">
        <source>Exceptions thrown:</source>
        <target state="translated">Excepciones devueltas:</target>
        <note>This is usually precedes by TestAssembly_AssemblyDiscoveryFailure message, and preceded by list of exceptions thrown in a test discovery session.</note>
      </trans-unit>
      <trans-unit id="FailedFetchAttributeCache">
        <source>Failed to get attribute cache. Ignoring attribute inheritance and falling into 'type defines Attribute model', so that we have some data.</source>
        <target state="translated">No se pudo obtener la memoria caché de atributos. Se omitirá la herencia del atributo y se entrará en "type define Attribute model", de modo que tengamos algunos datos.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="FailedToGetCustomAttribute">
        <source>Getting custom attributes for type {0} threw exception (will ignore and use the reflection way): {1}</source>
        <target state="translated">Al obtener atributos personalizados para el tipo {0} se produjo una excepción (se omitirá y se usará la forma de reflexión): {1}</target>
        <note>{0}: Attribute full type name.
{1}: Exception description</note>
      </trans-unit>
      <trans-unit id="OlderTFMVersionFound">
        <source>An older version of MSTestV2 package is loaded in assembly, test discovery might fail to discover all data tests if they depend on `.runsettings` file.</source>
        <target state="translated">Hay una versión anterior del paquete MSTestV2 cargada en el ensamblado. Es posible que la detección de pruebas no detecte todas las pruebas de datos si dependen del archivo ".runsettings".</target>
        <note></note>
      </trans-unit>
      <trans-unit id="UTA_UserCodeThrewNullValueException">
        <source>The called code threw an exception that was caught, but the exception value was null</source>
        <target state="translated">El código llamado produjo una excepción que se detectó, pero el valor de la excepción era null</target>
        <note></note>
      </trans-unit>
      <trans-unit id="OlderTFMVersionFoundClassCleanup">
        <source>An older version of MSTestV2 package is loaded in assembly, test cleanup methods might not run as expected. Please make sure all your test projects references MSTest packages newer then version 2.2.8.</source>
        <target state="translated">Hay una versión anterior del paquete MSTestV2 cargada en el ensamblado. Es posible que los métodos de limpieza de pruebas no se ejecuten según lo esperado. Asegúrese de que todos los proyectos de prueba hacen referencia a paquetes MSTest más recientes que la versión 2.2.8.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="CannotExpandIDataSourceAttribute">
        <source>Exception occurred while expanding IDataSource rows from attribute on "{0}.{1}": {2}</source>
        <target state="translated">Se produjo una excepción al expandir las filas de IDataSource del atributo en "{0}.{1}": {2}</target>
        <note>{0}: TypeName with namespace, 
{1}: Method name,
{2}: CannotExpandIDataSourceAttribute_DuplicateDisplayName or CannotExpandIDataSourceAttribute_CannotSerialize</note>
      </trans-unit>
      <trans-unit id="CannotExpandIDataSourceAttribute_DuplicateDisplayName">
        <source>Display name "{2}" on indexes {0} and {1} are duplicate. Display names should be unique.</source>
        <target state="translated">El nombre para mostrar "{2}" en los índices {0} y {1} están duplicados. Los nombres para mostrar deben ser únicos.</target>
        <note>{0}, {1}: Zero based index if an element inside of an array
{2}: Test display name.</note>
      </trans-unit>
      <trans-unit id="CannotExpandIDataSourceAttribute_CannotSerialize">
        <source>Data on index {0} for "{1}" cannot be serialized. All data provided through "IDataSource" should be serializable. If you need to test non-serializable data sources, please make sure you add "TestDataSourceDiscovery" attribute on your test assembly and set the discovery option to "DuringExecution".</source>
        <target state="translated">No se pueden serializar los datos del {0} de índice para "{1}". Todos los datos proporcionados a través de "IDataSource" deben ser serializables. Si necesita probar orígenes de datos no serializables, asegúrese de agregar el atributo "TestDataSourceDiscovery" en el ensamblado de prueba y establezca la opción de detección en "DuringExecution".</target>
        <note>{0}: Zero based index if an element inside of an array,
{1}: Test name</note>
      </trans-unit>
    </body>
  </file>
</xliff>